tmcprototype/CentralNode/CentralNode/CentralNode.py

```diff
@@ -262,6 +292,7 @@ def init_device(self):
                 print(CONST.STR_ERR_MSG, except_occured)
                 self._read_activity_message = CONST.STR_ERR_MSG + str(except_occured)
 
+
```

Comment:
Why extra line break is added here?

Reply:
Removed it. The commit id is: https://github.com/ska-telescope/tmc-prototype/commit/dafa12fbac0e08db8ec3ec703afb99ba45db38c1